api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -156,3 +156,43 @@ enum IneligibleStatus {
   // Only applicable to members.
   INELIGIBLE_STATUS_EXPIRED = 4;
 }
+
+// Review is a review of an access list.
+message Review {
+  // header is the header for the resource.
+  teleport.header.v1.ResourceHeader header = 1;
+
+  // spec is the specification for the access list review.
+  ReviewSpec spec = 2;
+}
+
+// ReviewSpec is the specification for an access list review.
+message ReviewSpec {
+  // access_list is the name of the access list that this review is for.
+  string access_list = 1;
+
+  // review is the user who performed the review.
+  string reviewer = 2;
```

Comment:
Should we make this a list, just in case for a bit of future-proofing?

Reply:
Not a bad idea, will do!